packages/terra-data-grid/src/subcomponents/ColumnHeaderCell.jsx

```diff
@@ -222,6 +223,12 @@ const ColumnHeaderCell = (props) => {
     }
   };
 
+  const handleFocus = () => {
+    if (isResizable && !isResizeHandleActive) {
+      columnHeaderCellButtonRef.current?.focus();
+    }
+  };
```

Comment:
Instead of adding a new focus event listener, I would expect that we could leverage the useEffect logic to make this work properly.

Reply:
Removed.